client/knode/protocol/const.go

```diff
@@ -8,18 +8,21 @@ import (
 // Constants to match up protocol versions and messages
 const (
 	Kcoin1 = 1
+
+	// Official short name of the protocol used during capability negotiation.
+	ProtocolName = "kcoin"
 )
 
-// Official short name of the protocol used during capability negotiation.
-const ProtocolName = "kcoin"
+var (
```

Comment:
I think this would sit better in a struct/object

Reply:
I can do that.